tasks/pipeline.py

```diff
@@ -401,7 +401,7 @@ def wait_for_pipeline_from_ref(repo: Project, ref):
 
 
 @task(iterable=['variable'])
-def trigger_child_pipeline(_, git_ref, project_name, variable=None, follow=True):
+def trigger_child_pipeline(_, git_ref, project_name, variable=None, follow=True, timeout=7200):
```

Comment:
Is 7200 the expected value? The previous default was 2700 but I'm not sure which one is a typo for the other :sweat_smile:

Reply:
yes 7200 (=2h) is the default as equal to the job timeout duration. 2700 was 45min, and as you mentioned yesterday it was too short